src/pages/_app.tsx

```diff
@@ -21,6 +21,11 @@ const MyApp: AppType<{ session: Session | null }> = ({
           name="viewport"
           content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"
         />
+        <meta name="og:title" content="OpenPipe" />
+        <meta name="og:description" content="Tinker with the latest LLMs." />
```

Comment:
Maybe we should match the tagline you put on our landing page?

Title: "OpenPipe: Open-Source Lab for LLMs"
Description: "OpenPipe is a powerful playground for quickly optimizing performance, cost, and speed across models."

I think I like those better.

Reply:
Updated